ricecooker/classes/nodes.py

```diff
@@ -691,6 +710,44 @@ def validate(self):
         assert (
             self.role in ROLES
         ), "Assumption Failed: Role must be one of the following {}".format(ROLES)
+
+        assert (
+            self.grade_levels in levels.LEVELSLIST
```

Comment:
Need to conditionalize these assertions:

First check `if self.grade_levels is not None:`

Then you need to check each of the values if it is a list, and see if they are in the `levels.LEVELSLIST`.